docs/content/administration/config-cheat-sheet.en-us.md

```diff
@@ -107,6 +107,7 @@ In addition, there is _`StaticRootPath`_ which can be set as a built-in at build
 - `PREFIX_ARCHIVE_FILES`: **true**: Prefix archive files by placing them in a directory named after the repository.
 - `DISABLE_MIGRATIONS`: **false**: Disable migrating feature.
 - `DISABLE_STARS`: **false**: Disable stars feature.
+- `DISABLE_STAR_LISTS`: **false**: Disable star lists feature.
```

Comment:
What is the use case to disable this but keep stars enabled? Imho it's enough to be able to disable stars as whole and we don't need to disable every feature